build/common.props

```diff
@@ -13,166 +13,17 @@
-        https://github.com/auth0/auth0.net/blob/master/SECURITY-NOTICE.md#idtokenvalidator-public for more details.
-
-      Version 6.5.3
-      WARNING: If you generate tokens in your project via System.IdentityModel.Tokens.Jwt
-      please read the important notice at https://github.com/auth0/auth0.net/issues/300
-
-      - Upgraded System.IdentityModel.Tokens.Jwt to 5.5 to fix incompatible kid
-      - Upgraded Microsoft.IdentityModel.Protocols.OpenIdConnect to 5.5
-      - Add ClientId to VerifyEmailJobRequest
-      - Updated all test dependencies (xunit, FluentAssertions, .NET Test SDK)
-      - Removed unused Console Workbench project
-
-      Version 6.5.2
-      - UserClient.GetEnrollments now correctly passes user id.
-
-      Version 6.5.1
-      - User and role permissions endpoints in UsersClient and RolesClient paging fix.
-
-      Version 6.5.0
-      - Assembly is now strong-name-signed so it can be used by other strong-name-signed packages.
-      - NOTE: This is code signing only using a non-secret key. It is not authenticode or tamper protection.
-      - User and role permissions endpoints in UsersClient and RolesClient now correctly honoring paging.
-      - User model optional fields (CreatedAt, UpdatedAt, LastLogin) are now nullable.
-
-      Version 6.4.0
-      - TenantSettings lifetimes are now double not integer.
-      - Added various Guardian-related endpoints on UserClient.
-
-      Version 6.3.0
-      - Missing Tenant settings now available (device flow, Guardian MFA, Change Password, flags etc.
-
-      Version 6.2.0
-      - Added client_id to GetDeviceCredentials response
-      - Added various user properties to UserUpdateRequest
-
-      Version 6.1.0
-      - New user permission endpoints added to UsersClient
-      - New role permission endpoints added to RolesClient
-      - AuthenticationApiClient now implements IDisposable to dispose ApiConnection and HttpClient
-      - Added various new and missing properties to Resource Servers (ResourceServerBase)
-
-      Version 6.0.0
-      - New GuardianClient for managing /guardian endpoints
-      - New RolesClient for managing /roles endpoints
-      - PasswordChangeTicket now has IncludeEmailInRedirect and MailEmailAsVerified
-      - ApiConnection now has Dispose to dispose the HttpClient it creates
-      - ManagementApiClient now has Dispose to dispose the ApiConnection it creates
-      - XML documentation tweaks
-      - Dependencies updated
-
-      BREAKING CHANGES
-      See our migration guide at https://github.com/auth0/auth0.net/blob/prepare-6.0.0/docs-source/migrating.md
-
-      - All I*Client interfaces have been removed so adding endpoints is no longer breaking
-      - IManagementApi interface was removed so adding new clients is no longer breaking
-      - All non-paging GetAll methods have been removed
-      - DiagnosticsHeader/DiagnosticsComponent are no longer available
+      https://github.com/auth0/auth0.net/blob/master/CHANGELOG.md
```

Comment:
Replaced the release notes with a link to our changelog. This will replace the entire release notes here: https://www.nuget.org/packages/Auth0.AuthenticationApi/